felix/docker-image/Dockerfile.amd64

```diff
@@ -22,7 +22,8 @@ FROM ubuntu:focal as wgtool
 RUN apt-get update && \
```

Comment:
If we can now support multiple arches with one dockerfile, let's get rid of the symlink and move this real file into its place (and maybe add a comment saying it's a multi-arch dockerfile now) :tada:

Reply:
> If we can now support multiple arches with one dockerfile, let's get rid of the symlink and move this real file into its place (and maybe add a comment saying it's a multi-arch dockerfile now) 🎉

OK, I will update it.